rotkehlchen/api/v1/encoding.py

```diff
@@ -1253,3 +1258,41 @@ class AssetIconsSchema(Schema):
         validate=webargs.validate.OneOf(choices=('thumb', 'small', 'large')),
         missing='thumb',
     )
+
+
+class CurrentAssetsPriceSchema(Schema):
+    assets = fields.List(
+        AssetField(required=True),
+        required=True,
+        validate=webargs.validate.Length(min=1),
+    )
+    async_query = fields.Boolean(missing=False)
+
+
+def _validate_historical_assets_price_timestamp(
+        asset_timestamp: Tuple[Asset, Timestamp],
+) -> None:
+    """Check the given timestamp is equal or less than today at 00:00:00 (the
```

Comment:
As discussed in today's standup this limitation does not apply to cryptocompare (when they actually do have data for an asset).

So it makes no sense to generally limit all input due to coingecko. If coingecko does not have a price for a timestamp, that's fine. But cutting it at input validation will stop us from trying all other sources.